3.Tests/UI/StepDefinitions/CommonSteps.cs

```diff
@@ -13,6 +13,12 @@ public CommonSteps(ScenarioContext scenarioContext)
         _scenarioContext = scenarioContext;
     }
 
+    [AfterScenario]
```

Comment:
Please move this to a CommonHooks class, it's not a big change and that way the rest of the team will be able to use that hook

Reply:
@sebasonti-jalauni The session disposal is a hook that **every single test** has to execute after its execution has been completed, this is why I changed it to the common steps due to this class being the base class of every test that involves step definitions
Leaving it there will prevent having to add a `@<session-disposal>` tag on every test just for the teardown of the driver.
The tags for the teardown of created project/items will be added in the hooks folder and will be executed in the tests that require an specific action.

Reply:
Actually, if a hook doesn't receive any tag it will run for every scenario/feature, hence that won't be an issue. All of this is to have the code more organized. So, there is no need to mix hooks and steps on the same class.